cdci_data_analysis/flask_app/schemas.py

```diff
@@ -21,16 +21,17 @@ class Meta:
     query_status = fields.Str(
                         validate=OneOf(["done", "failed", "submitted"]),
                         description="",
-                        required=True
+                        required=False,
+                        missing=""
```

Comment:
this is not a valid value for this field, see OneOf above. It never makes sense for this to be ""

Reply:
Sure, then one of the possible values should be used, so a `query_status` is always returned

Reply:
it makes sense to return it when it is used by the clients. Is there really a meaningful default when no information on actual value is provided?
Instead, I would specialize QueryOut to cases when some things are expected and other when it is needed not to be. Gradually.

But to implement this "gradually" we can add switches, and remove/make-on-by-default them when it is sorted out.
https://github.com/oda-hub/dispatcher-app/pull/199

Or at least leave TODO comments in less important cases, like this one.